draft-packages/illustration/KaizenDraft/Illustration/Scene.tsx

```diff
@@ -3,7 +3,10 @@ import { Base, BaseProps } from "./Base"
 import { VideoPlayerProps, VideoPlayer } from "./Players/VideoPlayer"
 import { SubsetBecomesNever } from "./types"
 
-export type SceneProps = Pick<BaseProps, "alt" | "classNameAndIHaveSpokenToDST">
+export type SceneProps = Pick<
+  BaseProps,
+  "alt" | "classNameAndIHaveSpokenToDST"
+> & { enableAspectRatio?: boolean }
 export type AnimatedProps = { isAnimated?: true } & Pick<
   VideoPlayerProps,
   "loop" | "autoplay"
```

Comment:
@ActuallyACat let me know if it's worth adding this to the animated versions

Reply:
I'd probably defer to @katsambarcus, I don't _think_ the animated scene illustrations are fairly static and unlikely to change in the future.

Reply:
There is some likelihood that the Brand Moments could, in the future, swap out the animation with another depending on the level of sentiment For example, a super positive brand moment vs subtle positive brand moment — i.e. someone jumping up and down vs a chilled guy swinging in a hammock).

We haven't developed any more animations than what we currently have, but there is a possibility we could go down this path in the future. I'd suggest we build in the aspect ration container around the animations, if it's possible.